src/routes/collection/+page.svelte

```diff
@@ -0,0 +1,474 @@
+<svelte:head>
+    <title>Lovat Collection | Lovat</title>
+</svelte:head>
+
+
+<section class="front-page">
+    <div class="content-wrapper">
+        <div class="header"> 
+            <img class="icon" src="/assets/img/collection-icon.png" alt="Screenshot of Lovat's Collection icon.">
```

Comment:
Icon looks good! The glow clips into the navigation bar a bit, might want to play with the blur radius.